src/databricks/labs/ucx/azure/resources.py

```diff
@@ -94,6 +115,18 @@ class AzureRoleAssignment:
     role_name: str
 
 
+@dataclass
+class AccessConnector:
+    id: str
```

Comment:
```suggestion
    id: AzureResource
```
so that we can extract resource group and subscription id easily.